server/protocols/jmap/src/main/java/org/apache/james/jmap/model/MessageContentExtractor.java

```diff
@@ -67,12 +91,8 @@ private MessageContent parseMultipartMixed(Multipart multipart) throws IOExcepti
         List<Entity> parts = multipart.getBodyParts();
         if (! parts.isEmpty()) {
             Entity firstPart = parts.get(0);
```

Comment:
We can maybe stream, filter with instanceOf, then findFirst ?

It will make implementation more robust and actually allow the body text message to not be the first ?